src/front/macOS/kDrive/UI/Controllers/MainWindow/Onboarding/Steps/OnboardingStepViewController.swift

```diff
@@ -0,0 +1,88 @@
+/*
+ Infomaniak kDrive - Desktop
+ Copyright (C) 2023-2025 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Cocoa
+import Combine
+import kDriveCoreUI
+
+open class OnboardingStepViewController: NSViewController {
+    private static let minContainerWidth: CGFloat = 200
+    private static let maxContainerWidth: CGFloat = 450
+
+    public let containerView = NSView()
+    public let titleLabel = NSTextField(labelWithString: "")
+    public let descriptionLabel = NSTextField(labelWithString: "")
+    public let primaryButton = BorderedProminentButton()
+    public let secondaryButton = BorderlessButton()
+
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+        setupComponents()
+    }
+
+    private func setupComponents() {
+        containerView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
```suggestion
        containerView.translatesAutoResizingMaskIntoConstraints = false
```
would be more readable.